packages/vm/src/cache.rs

```diff
@@ -182,6 +198,24 @@ where
         self.inner.lock().unwrap().stats
     }
 
+    pub fn pinned_metrics(&self) -> PinnedMetrics {
```

Comment:
As this is public now, we also should have `PinnedMetrics` and `PerModuleMetrics` exported in lib.rs